jobs/db_store.go

```diff
@@ -338,11 +348,32 @@ func (s *DatabaseStore) scanParams(method string, params []byte) (interface{}, e
 	}
 }
 
+func (s *DatabaseStore) scanResult(method string, result []byte) (interface{}, error) {
+	if result == nil {
+		return nil, nil
```

Comment:
Good question -- otherwise `if scannedresult == nil` might not work (differently typed nils are not equal)?